index.js

```diff
@@ -24,6 +24,7 @@ const defaults = {
   input: '.combobox',
   list: '.listbox',
   options: '.listbox .option',
+  optgroup: '.listbox .optgroup[data-label]',
```

Comment:
I think opt groups are far too uncommon to have this plugin default to an opt group.  It by default should be turned off and only if a `new Combobox` is instantiated with the `optgroup` configuration option should the functionality be turned on